regulations/static/regulations/css/less/fonts.less

```diff
@@ -3,3 +3,50 @@
  ==========================================================================
  Place all @font-face rules in fonts.less
  */
+
+ @font-face {
+   font-family: 'Source Sans Pro';
+   font-style: normal;
+   font-weight: 300;
+   src: url("../font/SourceSansPro-Light.ttf");
+ }
+
+ @font-face {
+   font-family: 'Source Sans Pro';
+   font-weight: 400;
+   src: url("../font/SourceSansPro-Regular.ttf");
+ }
+
+ @font-face {
+   font-family: 'Source Sans Pro';
+   font-style: italic;
+   font-weight: 400;
+   src: url("../font/SourceSansPro-Italic.ttf");
+ }
+
+ @font-face {
+   font-family: 'Source Sans Pro';
+   font-weight: 600;
+   src: url("../font/SourceSansPro-Semibold.ttf");
+ }
+
+ @font-face {
+   font-family: 'Source Sans Pro';
+   font-style: italic;
+   font-weight: 600;
+   src: url("../font/SourceSansPro-SemiboldItalic.ttf");
+ }
+
+ @font-face {
+   font-family: 'Merriweather';
+   font-style: normal;
+   font-weight: 400;
+   src: url("../font/Merriweather-Regular.ttf");
+ }
+
+ @font-face {
+   font-family: 'Merriweather';
+   font-style: italic;
+   font-weight: 400;
+   src: url("../font/Merriweather-Italic.ttf");
+ }
```

Comment:
Generally, browsers don't download fonts until they are used on a page.
More info: http://stackoverflow.com/questions/13249773/when-do-web-fonts-load-and-can-you-pre-load-them